src/ocf_seq_cutoff.c

```diff
@@ -7,17 +7,26 @@
 #include "ocf_cache_priv.h"
 #include "ocf_priv.h"
 #include "ocf/ocf_debug.h"
+#include "utils/utils_cache_line.h"
 
 #define SEQ_CUTOFF_FULL_MARGIN \
                 (OCF_TO_EVICTION_MIN + OCF_PENDING_EVICTION_LIMIT)
 
-static inline bool ocf_seq_cutoff_is_on(ocf_cache_t cache)
+/**
+ * @brief Check if sequential cutoff should be activated or not
+ *
+ * @param cache Cache instance
+ * @param clines Cachelines number that could be inserted with current request
+ *
+ * @return true if seq cutoff should be activated, false otherwise
+ */
+static inline bool ocf_seq_cutoff_is_on(ocf_cache_t cache, uint64_t clines)
```

Comment:
This function could take only one argument `req` and get number of clines from `req->core_line_count`. That would simplify this function call in `ocf_core_seq_cutoff_check()`.